service/store/store.go

```diff
@@ -316,8 +316,12 @@ func (s *Store) SetAwaitingProposalCid(id auction.BidID) error {
 	if err != nil {
 		return err
 	}
+	if b.Status == BidStatusAwaitingProposal {
+		log.Infof("bid %s already in '%s', duplicated message?", b.ID, b.Status)
```

Comment:
I *guess* this is what happening for the below error we saw from auctioneer, because with the automatic re-publishing it's very likely the message being delivered more than once, causes the status being already at `BidStatusAwaitingProposal`.

> publishing win in auction 01fc8sg8fysj5mhzav6ex3ta7g; bidder 12D3KooWKjqMa8PijM3gLhtDTgGtaRNKitoiz9U8XVS7YLndRe8H returned error: setting awaiting proposal cid: bid must have status 'submitted'

Reply:
Makes sense. How about using `Debug` here instead of `Info`? If re-publishing is an _expected_ thing in the system, we probably don't want to pollute the bidbot output with these by default.

Reply:
Not sure... we should minimize re-publishing, including that possible bug in pubsub-rpc library. I'd vote for some visibility here.